tests/functional/parser/test_parser.py

```diff
@@ -53,6 +54,33 @@ def handle_no_envelope(event: Dict, _: LambdaContext):
     handle_no_envelope(dummy_event["payload"], LambdaContext())
 
 
+def test_pydanticv2_validation():
+    from pydantic import __version__
+
+    version = __version__.split(".")
+
+    # GIVEN pydantic v2 version
+    if int(version[0]) == 2:
```

Comment:
I would prefer a pytest.skip and detent the other code.

```suggestion
    if version[0] != "2":
        pytest.skip("test only for pydantic v2.")
```

or maybe better as a fixture?

```
@pytest.fixture
def pydanticv2_only():
    from pydantic import __version__

    version = __version__.split(".")
    if version[0] != "2":
        pytest.skip("pydanticv2 test only")

@pytest.mark.usefixture("pydanticv2_only")
def test_pydanticv2_validation():
    ...

Reply:
Hi @Wurstnase! Thanks a lot for suggesting this. It makes perfect sense to use a fixture for this because we can introduce new tests for v2 without repeating too much code.